vrp/vrp.py

```diff
@@ -85,11 +85,21 @@ def distance_callback(from_index, to_index):
 
     # Define cost of each arc.
     routing.SetArcCostEvaluatorOfAllVehicles(transit_callback_index)
+    
+    # to not drop routes removes limiting constraints
+    dropRoutes = False
+    if not dropRoutes:
```

Comment:
I'm a bit confused about this section. Doesn't look like the `dropRoutes` variable gets reassigned so is this check strictly necessary?

Reply:
I mostly wanted to not hard-code parameters in case Esko doesnt like this, we could easily change drop routes to be true or just delete the additions